lib/modules/manager/helmfile/artifacts.ts

```diff
@@ -59,9 +67,28 @@ export async function updateArtifacts({
         constraint: config.constraints?.kustomize,
       });
     }
-    await exec(`helmfile deps -f ${quote(packageFileName)}`, {
+
+    const cmd: string[] = [];
+    const doc = parseDoc(newPackageFileContent);
+
+    const repositories = getRepositories(doc);
+
+    for (const value of repositories.filter(isOCIRegistry)) {
+      const loginCmd = generateRegistryLoginCmd(
+        value.name,
+        `https://${value.url}`,
+        value.url.replace(regEx(/\/.*/), '')
```

Comment:
cannot use replaceUrlPath because of value.url format is like ghcr.io/helm-charts.

Reply:
please add a comment that this extracts the hostname. it's not obvious to see from code